src/components/Signin.jsx

```diff
@@ -0,0 +1,281 @@
+import React,{useState} from 'react';
+import undraw_login from '../components/assets/images/undraw_Login.svg';
+import styled from 'styled-components';
+
+
+const Content = styled.div`
+  width: 70%;
+`;
+
+const Button = styled.button`
+  width:${props => props.width};
+  margin-left:${props=>props.marginLeft};
+`;
+
+
+const Seperator = styled.div`
+  margin:40px;
+  h2 { 
+    width:100%; 
+    text-align:center; 
+    border-bottom: 5px solid gray; 
+    line-height:0.1em; 
+    margin:10px 0 20px;
+  } 
+  h2 span { 
+    background:#fff; 
+    padding:0 10px; 
+  }
+
+`;
+
+
+const DisplayForLinks = styled.div`
+  display: inline;
+  list-style:none;
+  text-align: center;
+  width: 100%;
+  margin: auto;
+  padding: 0 50px;
+  margin-top: 6px;
+  justify-content: space-around;
+
+  li{
+    margin:10px;
+  }
+  a{
+      width: 90%;
+  }
+
+  @media screen and (max-width: 650px) {
+    .col {
+      width: 100%;
+      margin-top: 0;
+    }
+  }
+`;
+
+const AuthLink = styled.a`
+  background-color:${props => props.bgColor} ;
+  color: ${props => props.color};
+  border : ${props => props.border};
+
+  &:hover{
+    background-color:${props => props.hoverBgColor};
+    color:${props => props.hoverColor};
+  }
+  i{
+    font-size:${props => props.IconSize};
+    color:${props => props.IconColor};
+    float:${props => props.IconFloat};
```

Comment:
Why do you have to use float property.

Reply:
becuase if i dont use the icon is coming above the content and not beside the content, so i used float.